packages/manager/modules/pci/src/projects/project/data-processing/notebooks/data-processing-notebooks.controller.js

```diff
@@ -0,0 +1,72 @@
+import {
+  isNotebookRunning,
+  getClassFromStatus,
+} from '../data-processing.utils';
+import { DATA_PROCESSING_GUIDE_URL } from '../data-processing.constants';
+
+export default class {
+  /* @ngInject */
+  constructor(
+    $state,
+    CucCloudMessage,
+    dataProcessingService,
+    ovhManagerRegionService,
+    atInternet,
+  ) {
+    this.$state = $state;
```

Comment:
Do you still need to inject $state ? Looks like it's not used here.

Reply:
Good point. Fix'd !